wsu-people-directory.php

```diff
@@ -14,8 +14,16 @@
 	die;
 }
 
-// Flush rewrite rules on activation or deactivation.
-register_activation_hook( __FILE__, 'flush_rewrite_rules' );
+
+register_activation_hook( __FILE__, 'wsuwp_people_directory_activate' );
```

Comment:
Ah, I remember coming across a few articles about the network activation deal when I first looked into this piece.

But you're right - we can just ditch it. The post type's `rewrite` argument is set to `false`, so there's no need to flush the rewrite rules on activation (duh!).